util/src/main/resources/tech/pegasys/teku/util/config/less-swift.yaml

```diff
@@ -0,0 +1,161 @@
+# Teku specific Swift preset:
+# Made for quick acceptance tests.
+# Copy of the minimal preset but
+# SECONDS_PER_SLOT is 1 instead of 6 and
+# SLOTS_PER_EPOCH is 4 instead of 8
+# MIN_GENESIS_ACTIVE_VALIDATOR_COUNT is 4 instead of 64
+
+CONFIG_NAME: "less-swift"
```

Comment:
Why did we need this instead of just swift which is reasonable sane now or just minimal?

Reply:
Because we needed both short a really short genesis delay, and 6 second slots. Swift has 3 second slots, and minimal has 300 seconds genesis delay.

Reply:
and now we need a SHARD_COMMITTEE_PERIOD of 0.